### DISCUSSION TITLE
제페토에서 스크린샷 코드 제공해주신것처럼 영상찍는 기능도 만들수 있을까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#656] [5. Q&A]
제페토 내에서 스크린샷 기능을 제공하시고, 이번에 코드로 해당 기능을 해금해 주신것 처럼,

영상을 찍어서 피드에 업로드하는 기능을, 혹은 로컬 저장소에 저장하는 기능을 넣을 수 있을까요?

잘부탁드립니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 크리에이터님~ 동영상 촬영 기능은 준비 중에 있습니다 😊
업데이트 시 안내 드리겠습니다.
감사합니다.

**Reply by ted002** [reply]
와 감사합니다!

**Reply by ted002** [reply]
와 동영상 촬영 가능! 감사합니다!
